includes/blocks/class-sensei-lesson-blocks.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
Written this class as part of my investigation of adding the blocks to the lesson page. Happy to remove it since it is unused.